setup.py

```diff
@@ -40,16 +40,12 @@
 
 INSTALL_REQUIRES = [
     'PyPDT>={0}'.format(PYPDT_MIN_VERSION),
-    'numpy>=1.11.0,<1.12.0;python_version<"2.7"',
     'numpy>={0};python_version>="2.7"'.format(NUMPY_MIN_VERSION),
-    'pandas==0.16.2;python_version<"2.7"',
     'pandas;python_version>="2.7"',
-    'matplotlib<1.5;python_version<"2.7"',
     'matplotlib>2.0.0,<2.1;python_version>="2.7"',
```

Comment:
Am I reading this correctly that the latest supported `matplotlib` version is 2.1?

Reply:
This one I also stumbled on. I wonder if that's necessary. Let's see what @brovercleveland says as I seem to remember he added this when working on the visual module … It would be good to get rid of this requirement for sure!

Reply:
I tried with higher versions of matplotlib and yes the MplPlotter module of skhep doesn't work.

Reply:
Thanks @marinang. We should then create an issue to see if the requirement can be removed somehow …

Reply:
Issue added - https://github.com/scikit-hep/scikit-hep/issues/68.